internal/tiltfile/tiltfile_state.go

```diff
@@ -633,6 +686,54 @@ func (s *tiltfileState) assembleK8sV2() error {
 	return nil
 }
 
+// format is <name:required>:<kind:optional>:<namespace:optional>
+func selectorFromString(s string) (k8sObjectSelector, error) {
```

Comment:
inconsistency: this format and the code below omit the "apiVersion" arg, while the docs and the error message in this func ("between 1 and 4 parts") indicate that `apiVersion` is an acceptable arg.

Reply:
(note you may have to change `selectorStringFromK8sEntity` as well)